### ISSUE TITLE
ViaVersion 4.5.0 | command execute bug

### ISSUE THREAD

**Issue by Sakaki-Aruka** [ViaVersion/ViaVersion#3163]
### `/viaversion dump` Output

https://dump.viaversion.com/7de5e1197ad717b425fcbce8151b403f98eede899d11bb5ec6992512b1d92176

### Console Error

```
Put the paste.gg link or text here.

14:13:44 [警告] [ViaVersion] ERROR IN Protocol1_19_3To1_19_1 IN REMAP OF CHAT_COMMAND (0x04)
14:13:44 [警告] [ytshiyugh] -> UpstreamBridge - could not decode packet! com.viaversion.viaversion.exception.InformativeException: Please report this on the Via support Discord or open an issue on the relevant GitHub repository
Packet Type: CHAT_COMMAND, Type: BitSet, Data: [Pair{Type|String, say hello}, Pair{Type|Long, 1670649224713}, Pair{Type|Long, -5756731238082055048}, Pair{Type|VarInt, 0}, Pair{Type|Boolean, false}, Pair{Type|PlayerMessageSignature Array, [Lcom.viaversion.viaversion.api.minecraft.PlayerMessageSignature;@5cd9da4}, Pair{Type|PlayerMessageSignature, null}], Source 0: com.viaversion.viaversion.protocols.protocol1_19_3to1_19_1.Protocol1_19_3To1_19_1$7 (Anonymous), Packet ID: 4
14:13:44 [情報] [ytshiyugh] disconnected with: DecoderException : com.viaversion.viaversion.exception.InformativeException: Please report this on the Via support Discord or open an issue on the relevant GitHub repository
Packet Type: CHAT_COMMAND, Type: BitSet, Data: [Pair{Type|String, say hello}, Pair{Type|Long, 1670649224713}, Pair{Type|Long, -5756731238082055048}, Pair{Type|VarInt, 0}, Pair{Type|Boolean, false}, Pair{Type|PlayerMessageSignature Array, [Lcom.viaversion.viaversion.api.minecraft.PlayerMessageSignature;@5cd9da4}, Pair{Type|PlayerMessageSignature, null}], Source 0: com.viaversion.viaversion.protocols.protocol1_19_3to1_19_1.Protocol1_19_3To1_19_1$7 (Anonymous), Packet ID: 4 @ io.netty.handler.codec.MessageToMessageDecoder:98
```

※警告 means "Warn"
※情報 means "Info"

### Bug Description

![image](https://user-images.githubusercontent.com/99568054/206830561-fe9a3e16-cff0-4726-b6d7-717eaf8363c9.png)

When I execute a command that "/say hello" , i kicked from server and saw the logs.

### Steps to Reproduce

1. join a server (1.19.2 with ViaVersion(ver latest)) used Java Edition 1.19.3
2. execute a command "/say hello"
3. kicked from a server, and logs displayed.

### Expected Behavior

i can't think deeper.

### Additional Server Info

Env:
 - Proxy : bungeecord (latest)
 - Proxy-plugin : ViaVersion(latest) , floodgate(latest) , Geyser(latest)
 - Server : spigot(1.19.2)

 - Client Env : Java Edition ver 1.19.3

### Checklist

- [X] Via plugins are only running on **EITHER** the backend servers (e.g. Paper) **OR** the proxy (e.g. BungeeCord), **not on both**.
- [X] I have included a ViaVersion dump.
- [X] If applicable, I have included a paste (**not a screenshot**) of the error.
- [X] I have tried the latest build(s) from https://ci.viaversion.com/ and the issue still persists.

**Comment by Sakaki-Aruka** [comment]
add info
build numbers:
 - ViaVersion : release 4.5.0
 - bungeecord : build 1672
 
 (- geyser-bungeecord : build 1252)
 (- floodgate : build 73)

**Comment by Jaren8r** [comment]
This should've been fixed by 2a49fc0. Have you tried the latest dev build?

**Comment by Sakaki-Aruka** [comment]
oh i din't check that.
sorry.

when i use ViaVersion build-663, the bug was fixed like you said.
Sorry to waste your time. And thank you.
